src/ops/storages/postgres.rs

```
@@ -159,6 +159,9 @@ fn bind_value_field<'arg>(
                     builder.push_bind(sqlx::types::Json(v));
                 }
             },
+            BasicValue::UnionVariant { tag_id, value } => {
+                builder.push_bind(sqlx::types::Json(serde_json::json!([tag_id, value])));
```

For the target storage, the value is consumed by an external system and doesn't need to be read back, so we only to export the value without `tag_id`. 

We can serialize a `TypedValue` to JSON (similar to the last branch a few lines below) .

I know it'll introduce difficulty for the implementation of `from_pg_value()` below - but actually that one will be deprecated soon, so we don't need to support union type for it (background: https://github.com/cocoindex-io/cocoindex/issues/504; currently all examples are not using it) - we can raise an error when encountering union type there.